packages/smart-account/src/SmartAccount.ts

```diff
@@ -587,18 +590,19 @@ class SmartAccount extends EventEmitter {
     if (updateImplTrx.data != '0x') {
       transactions.unshift(updateImplTrx)
     }
-    const response = await aaSigner.sendTransactionBatch(transactions, false)
+    const response = await aaSigner.sendTransactionBatch(transactions, transactionBatchDto.vGasLimit, transactionBatchDto.gasLimit, false)
```

Comment:
for batch you are sending verificationGasLimit and gasLimit as custom but not for regular sendTransaction looks like its just verification gasLimit